kani-compiler/src/codegen_cprover_gotoc/compiler_interface.rs

```diff
@@ -81,6 +81,10 @@ impl CodegenBackend for GotocCodegenBackend {
         provide::provide_extern(providers);
     }
 
+    fn print_version(&self) {
```

Comment:
This doesn't seem to be used anywhere.

Reply:
In theory this is a function that the compiler uses to print the codegen version. Note that it is part of the implementation of the `CodegenBackend` trait.

There is a limitation in the compiler today where this doesn't get invoked for codegen backend that was configured via the driver, but I figured we might as well implement it and once this issue gets fixed, we get our version printed correctly.